executor/common.h

```diff
@@ -658,7 +658,8 @@ static void loop(void)
 			close(kOutPipeFd);
 #endif
 			execute_one();
-#if SYZ_HAVE_CLOSE_FDS && !SYZ_THREADED
+#if !SYZ_EXECUTOR && SYZ_HAVE_CLOSE_FDS && !SYZ_THREADED
+			// Executor's execute_one has already called close_fds.
```

Comment:
Would it be simpler if we didn't call `close_fds()` in `execute_one()`, but only here?
We'd get less code and ifdefs then.

Reply:
I think it's called in execute_one before dumping remote coverage on a purpose, see https://github.com/google/syzkaller/commit/5c51045d28eb1ad9465a51487d436133ce7b98d2
```
+       // Also close all USB emulation descriptors to trigger exit from USB
+       // event loop to collect coverage.
```